articles/azure-monitor/faq.yml

```diff
@@ -1080,6 +1080,21 @@ sections:
 
   - name: Container insights
     questions:
+      - question: |
+         Why do I see info banner "You don't have right cluster permissions which may restrict your access to the feature. Please reach-out to your cluster-admin to get the right permissions"?
```

Comment:
**warning banner**

also I updated the banner text to:

"You do not have the right cluster permissions which will restrict your access to Container Insights features. Please reach out to your cluster admin to get the right permissions."